config/warren_consumers.yml

```diff
@@ -1,6 +1,7 @@
 ---
 event_consumer:
   desc: Takes events off the queue and records them in the warehouse
+  worker_count: 1
```

Comment:
Maybe add a comment explaining why worker_count is 1. Could reference this story to, and/ or add a note in Sabrines decision making doc for future reference

Reply:
I have added a comment above the line and updated README with a worker_count section.

Reply:
I have added a new page [Use single worker thread for Unified Warehouse and Event Warehouse consumers](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Use+single+worker+thread+for+Unified+Warehouse+and+Event+Warehouse+consumers) under [Logging Infrastructure Decisions](https://ssg-confluence.internal.sanger.ac.uk/display/PSDPUB/Logging+Infrastructure+Decisions) as you suggested.